src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/combination.yml

```diff
@@ -39,6 +39,18 @@ admin_products_combinations_edit_combination:
   requirements:
     combinationId: \d+
 
+admin_products_combinations_bulk_edit_combination:
+  path: /combinations/{combinationId}/bulk-edit
+  methods: [ PATCH ]
```

Comment:
If we submit form with POST, all the disabled inputs are prefilled with empty values, which then breaks the form constraints, so PATCH seems to be the option to go.